### DISCUSSION TITLE
Could one endpoint provides multiple ML services?

### DISCUSSION THREAD

**Discussion by ShelbyHero** [bentoml/BentoML#4276] [Q&A] [answered]
I want to deploy multiple machine learning models on BentoML, but I want to deploy them to the same endpoint. That is, a endpoint provides multiple independent ML services. For example,I want to deploy model1 and model2 on the sever, model1 use the "localhost:5000/model1", and model2 use the "localhost:5000/modelA2" . As far as I know, I could make model1 use the "localhost:5000/model1" and make model2 use the"localhost:5001/model2 ", but that's not what I want. Can BentoML do this?Thanks for the answer!

**Comment by parano** [comment, marked as answer]
Hi @ShelbyHero , yes BentoML can do that - simplify define multiple runners and multiple APIs: https://docs.bentoml.com/en/latest/concepts/service.html#service-apis

**Reply by ShelbyHero** [reply]
Thanks! Can I run multiple services at the same time when using the same endpoint ?I can run one service with the cmd command now.Please tell me what I need to do~
